### ISSUE TITLE
❓ Question: How can i use useObservable to create a subject

### ISSUE THREAD

**Issue by ShravanSunder** [crimx/observable-hooks#48]
Hi! I'm currently learning RxJs and using it with react.  

` const flag$ = useObservable(pluckFirst, [props.flag])`

I was wondering how i can use the above to create a subject?  I wanted to do it while staying in react.  Do i have to create a separate subject resource thats outside of react?

thanks for the wonderful library

**Comment by crimx** [comment]
Yes you can! "[Do whatever you want with Observables](https://observable-hooks.js.org/)" is one of the main features of observable-hooks.

If you want to create Subject from dependencies, you can do this:

```ts
const flag$ = useObservable(
  inputs$ => inputs$.pipe(
    map(([flag]) => flag),
    share()
  ),
  [props.flag]
)
```

Or if you just want to create a Subject out of thin air, you can do this:

```ts
const subject$ = useObservable(() => new Subject())
```
